classes/class-uagb-block-helper.php

```diff
@@ -1511,6 +1527,9 @@ public static function uag_get_background_obj( $bg_obj, $css_for_pseudo = '' ) {
 						if ( 'none' === $overlay_type && '' !== $bg_img ) {
```

Comment:
We can remove the below block of code with this update.

```suggestion
						if ( ( 'none' === $overlay_type || 'image' === $overlay_type ) && '' !== $bg_img ) {
```

Reply:
updated change and removed the below code as per suggestion.